### PR TITLE
Kubeone apidoc regen

### PR DESCRIPTION
**Pull request by kron4eg**
* Kubeone apidoc regen
* kubeone registryConfiguration docs

### CODE REVIEW

File: content/kubeone/master/infrastructure/custom_registry.en.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
++++
+title = "Custom Registry"
+date = 2020-10-27T12:00:00+02:00
+weight = 4
++++
+
+## RegistryConfiguration
+
+It's possible to configure the central docker image registry to pull images from using `registryConfiguration` config.
```
Comment (xrstf) [inline content/kubeone/master/infrastructure/custom_registry.en.md:9]: > pull images from using

**Review by xrstf — CHANGES_REQUESTED** [review]
One minor typo

**Review by xrstf — APPROVED** [review]
/approve

**Comment by kron4eg** [comment]
/retest
/hold

hold until the upstream PR https://github.com/kubermatic/kubeone/pull/1148 is merged

**Pull request merged**